ai-services/app.py

```
@@ -1,78 +1,29 @@
-                    }
-                ],
-                "metadata": {
-                    "prompt": request.prompt,
-                    "options": request.options,
-                    "generationTime": "0.5s"
-                }
-            }
-        }
-    except Exception as e:
-        raise HTTPException(status_code=500, detail=str(e))
+from ai_services.main import app, start
```

Ruff (0.8.2)

`ai_services.main.app` imported but unused

Remove unused import: `ai_services.main.app`

(F401)